AddSsoDotnet3/Components/GetUserProfile.razor

```diff
@@ -0,0 +1,98 @@
+﻿@using System.IO
+@using Azure.Core
+@using Microsoft.Graph
+@inject TeamsFx teamsfx
+@inject TeamsUserCredential teamsUserCredential
+
+<div>
+    <h2>Get the user's profile</h2>
+    @if (NeedConsent)
+    {
+        <p>Click below to authorize this app to read your profile photo using Microsoft Graph.</p>
+        <FluentButton Appearance="Appearance.Accent" Disabled="@IsLoading" @onclick="ConsentAndShow">Authorize</FluentButton>
+    }
+    else if (!string.IsNullOrEmpty(@ErrorMessage))
+    {
+        <div class="error">@ErrorMessage</div>
+    }
+    else if (Profile != null)
+    {
+        <div>Hello @Profile.DisplayName</div>
+    }
+</div>
+
+@code {
+    [Parameter]
+    public string ErrorMessage { get; set; }
+
+    public bool IsLoading { get; set; }
+    public bool NeedConsent { get; set; }
+    public User Profile { get; set; }
+
+    private readonly string _scope = "User.Read";
+
+    protected override async Task OnInitializedAsync()
+    {
+        IsLoading = true;
+        if (await HasPermission(_scope))
```

Comment:
Can we show profile directly? why do we need to check permission first.

Reply:
I have tried that we can show it directly, but I think we may follow SSO enabled tab template here.